### PR TITLE
Sp 94 pick up path from env

### PR DESCRIPTION
**Pull request by BenStarmerSmith**
## Proposed changes

Added a PR to gather the desktop folder automatically, accounting for customers with a different country setup on Ubuntu.

## Checklist

Before posting a PR ensure that from each of the below categories **AT LEAST ONE BOX HAS BEEN CHECKED**. If more than one category is applicable then more can be checked. Also ensure that the proposed changes have been filled out with relevant information for reviewers.

## Tests

- [ ] No tests required to be added. (For small changes that will be tested by CI/CD infrastructure).
- [ ] Added/Modified automated and PhantomHand CI tests (if a new class is added (Python or C++), the interface of that class must be unit tested).
- [x] Manually tested in simulation (if simulation specific or no hardware required to test the functionality). 
- [ ] Manually tested on hardware (if hardware specific or related).

## Documentation

- [x] No documentation required to be added.
- [ ] Added documentation (For any new feature, explain what it does and how to use it. Write the documentation in a relevant space, e.g. Github, Confluence, etc).
- [ ] Updated documentation (For changes to pre-existing features mentioned in the documentation).

### CODE REVIEW

File: ansible/roles/products/common/resources/templates/scripts/setup-hand.j2 [hunk 1]
```diff
@@ -23,7 +23,7 @@ if [ -f /tmp/id_rsa ]; then
 fi
 if {{ launch_hand | lower }}; then
     if {{ sim_hand | lower }}; then
-        roslaunch sr_robot_launch srhand.launch
+        roslaunch sr_robot_launch srhand.launch sim:=true
```
Comment (biofotis) [inline ansible/roles/products/common/resources/templates/scripts/setup-hand.j2:26]: Why sim true

File: ansible/playbooks/docker_deploy.yml [hunk 2]
```diff
@@ -24,8 +24,9 @@
         use_cyberglove: true
 
   roles:
-    - {role: installation/docker}
-    - {role: installation/nvidia-docker, when: nvidia_docker | bool}
-    - {role: products/hand-e/deploy, when: product == 'hand_e'}
-    - {role: docker/aws, when: use_aws|bool}
-    - {role: products/common/dolphin-icons, when: ansible_distribution_release|string == 'focal'}
+    - { role: installation/docker}
```
Comment (biofotis) [inline ansible/playbooks/docker_deploy.yml:27]: Why this change here?

File: ansible/playbooks/server_and_nuc_deploy.yml [hunk 3]
```diff
@@ -77,8 +77,9 @@
       become: yes
 
   roles:
+    - { role: products/common/get-system-variables }
     - { role: products/hand-e/server/deploy }
-    - { role: products/common/dolphin-icons, when: ansible_distribution_release|string == 'focal'}
+    - { role: products/common/dolphin-icons, when: ansible_distribution_release|string == 'focal' }
```
Comment (biofotis) [inline ansible/playbooks/server_and_nuc_deploy.yml:82]: Why this?

File: ansible/playbooks/teleop_deploy.yml [hunk 4]
```diff
@@ -112,6 +114,7 @@
       become: yes
 
   roles:
+    - { role: products/common/get-system-variables }
```
Comment (biofotis) [inline ansible/playbooks/teleop_deploy.yml:117]: Is this needed on the nuc?

File: ansible/roles/products/common/dolphin-icons/tasks/main.yml [hunk 5]
```diff
@@ -92,6 +92,13 @@
   changed_when: false
   with_items: "{{ file_list.files }}"
 
+- name: Create shadow_hand_launcher folder
```
Comment (biofotis) [inline ansible/roles/products/common/dolphin-icons/tasks/main.yml:95]: What is this?

**Review by biofotis — CHANGES_REQUESTED** [review]
Your branch is outdated already? Dont you use an up to date master branch?

File: ansible/roles/products/common/resources/templates/scripts/setup-hand.j2 (hunk 1 above)
Comment (BenStarmerSmith) [reply ansible/roles/products/common/resources/templates/scripts/setup-hand.j2:26]: Because this is in the conditional if sim_hand = true then... but it was always missing sim:=true so I fixed it here

**Review by bdenoun — APPROVED** [review]
Not 100% sure of what that does but seems OK

File: ansible/playbooks/teleop_deploy.yml (hunk 4 above)
Comment (biofotis) [reply ansible/playbooks/teleop_deploy.yml:117]: but is there a system variable on nuc for desktop? I remember we create a desktop folder at some point. Have you checked it on nuc?

File: ansible/roles/products/common/dolphin-icons/tasks/main.yml (hunk 5 above)
Comment (biofotis) [reply ansible/roles/products/common/dolphin-icons/tasks/main.yml:95]: but you are on dolphin role now? This should have been created many steps above. If not thats not the place to add it

File: ansible/playbooks/teleop_deploy.yml (hunk 4 above)
Comment (BenStarmerSmith) [reply ansible/playbooks/teleop_deploy.yml:117]: So after testing this branch on server_and_nuc (with nuc plugged into server) the branch executed fine with no errors. After execution I SSH'd onto the nuc and ran the xdg-user-dir DESKTOP which returned this screenshot (I'll pm it to you). So I believe this works as intended on the NUC and should be left there incase we ever need future system variables like these.

**Review by biofotis — APPROVED** [review]
